launcher/Application.cpp

```diff
@@ -306,16 +306,20 @@ Application::Application(int &argc, char **argv) : QApplication(argc, argv)
     }
     else
     {
-#ifdef LAUNCHER_LINUX_DATADIR
-        QString xdgDataHome = QFile::decodeName(qgetenv("XDG_DATA_HOME"));
-        if (xdgDataHome.isEmpty())
-            xdgDataHome = QDir::homePath() + QLatin1String("/.local/share");
-        dataPath = xdgDataHome + "/polymc";
-        adjustedBy += "XDG standard " + dataPath;
-#elif defined(Q_OS_MAC)
+#if !Launcher_PORTABLE || defined(Q_OS_MAC)
         QDir foo(FS::PathCombine(QStandardPaths::writableLocation(QStandardPaths::AppDataLocation), ".."));
         dataPath = foo.absolutePath();
-        adjustedBy += "Fallback to special Mac location " + dataPath;
+        adjustedBy += dataPath;
+
+#ifdef Q_OS_LINUX
+        // TODO: this should be removed in a future version
+        // TODO: provide a migration path similar to macOS migration
```

Comment:
The migration path should be added in a separate PR I think. This should be fine for now.

Reply:
Looks good!